### ISSUE TITLE
cannot complete npm install on ubuntu

### ISSUE THREAD

**Issue by Wingie** [mirumee/saleor#545]
```
$ npm install --no-optional                                                                                                [17:12:05]
npm ERR! Linux 3.16.0-77-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--no-optional"
npm ERR! node v4.4.7
npm ERR! npm  v2.15.8
npm ERR! code EBADPLATFORM

npm ERR! notsup Unsupported
npm ERR! notsup Not compatible with your operating system or architecture: fsevents@1.0.14
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: x64

npm ERR! Please include the following file with any support request:
npm ERR!     /home/wingston/scrap/saleor/npm-debug.log
[npm-debug.txt](https://github.com/mirumee/saleor/files/422645/npm-debug.txt)



```

**Comment by patrys** [comment]
Can you try after removing npm-shrinkwrap.json? I think it accidentally includes an OSX-specific package.

**Comment by patrys** [comment]
Should be fixed now on `master` and `v2016.07` branches.

**Comment by Wingie** [comment]
Thank you. Yes deleting the file fixes the issue.
